components/director/internal/domain/formationtemplate/resolver_test.go

```diff
@@ -179,8 +179,10 @@ func TestResolver_FormationTemplates(t *testing.T) {
 			TxFn:  txGen.ThatSucceeds,
 			First: &first,
 			FormationTemplateService: func() *automock.FormationTemplateService {
+				var nilStr *string
```

Comment:
You can extract `nilStr` into a global var instead of redeclaring it in every service.